grapher/Models/AccelGUI.cs

```diff
@@ -140,7 +140,8 @@ public void UpdateActiveSettingsFromFields()
                 combineMagnitudes = ApplyOptions.IsWhole,
                 modes = ApplyOptions.GetModes(),
                 args = newArgs,
-                minimumTime = driverSettings.minimumTime
+                minimumTime = driverSettings.minimumTime,
+                negativeMultipliers = driverSettings.negativeMultipliers
```

Comment:
What does this do? Can't we just allow the user to set negative multipliers rather than needing this setting? Or is it meant to safeguard?

Reply:
These are for directional multipliers - rename